lib/ipmi_delloem.c

```diff
@@ -944,19 +944,23 @@ ipmi_lcd_get_info_wh(struct ipmi_intf * intf)
 				sizeof(lcd_caps), &lcd_caps);
 		if (rc < 0) {
 			lprintf(LOG_ERR, "Error getting LCD capabilities.");
-			return -1;
+			ERR_EXIT;
 		} else if ((rc == 0xc1) || (rc == 0xcb)) {
 			lprintf(LOG_ERR, "Error getting LCD capabilities: "
 					"Command not supported on this system.");
 		} else if (rc > 0) {
 			lprintf(LOG_ERR, "Error getting LCD capabilities: %s",
 					val2str(rc, completion_code_vals));
-			return -1;
+			ERR_EXIT;
 		}
 		if (lcd_caps.number_lines > 0) {
 			memset(lcdstring, 0, IPMI_DELL_LCD_STRING_LENGTH_MAX + 1);
 			rc = ipmi_lcd_get_single_line_text(intf, lcdstring,
-					lcd_caps.max_chars[0]);
+			                                   lcd_caps.max_chars[0]);
```

Comment:
I accidentally also changed this line -- which was untouched by this patchset.  I'm going to undo this.